src/rom.c

```diff
@@ -3,10 +3,22 @@
 #include <fcntl.h>
 #include <sys/mman.h>
 #include <string.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <SDL2/SDL.h>
```

Comment:
Why is SDL2 included in `rom.c`? This module should be kept separate from everything non-ROM related, if possible.

Reply:
Agreed, was necessary for the rom "teardown". That is, if `munmap` fails, we need a teardown, but we discussed this above already.